games-fps/shockolate/shockolate-20180924.recipe

```diff
@@ -0,0 +1,52 @@
+SUMMARY="System Shock, but cross platform!"
+DESCRIPTION="Based on the source code for PowerPC released by Night Dive Studios, Incorporated.\
+You need contents of data folder from the original game copied into res/data folder."
+HOMEPAGE="https://github.com/Interrupt/systemshock"
+COPYRIGHT="2018 Interrupt"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/Interrupt/systemshock/archive/1b1c664188ce736a343913de32afe627e48e7bbb.zip"
+CHECKSUM_SHA256="29685f42d4dc0e1b6a608ac61ce8f09fdce379784af73400fb9cd64aa9b6d51d"
+SOURCE_DIR="systemshock-1b1c664188ce736a343913de32afe627e48e7bbb"
+PATCHES="shockolate_x86-20180924.patchset"
+ARCHITECTURES="!x86_gcc2 !x86_64 ?x86"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	systemshock$secondaryArchSuffix = $portVersion
+	cmd:systemshock$secondaryArchSuffix = $portVersion
+"
+
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libsdl2_2.0$secondaryArchSuffix
+	lib:libsdl2_mixer_2.0$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libsdl2_2.0$secondaryArchSuffix
+	devel:libsdl2_mixer_2.0$secondaryArchSuffix
+	"
+
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	"
+
+defineDebugInfoPackage systemshock$secondaryArchSuffix \
+	$binDir/systemshock
+
+BUILD()
+{
+	cmake . $cmakeDirArgs
+	make $jobArgs systemshock
+}
+
+INSTALL()
+{
+	mkdir -p $binDir
+	cp systemshock $binDir
+	strip $binDir/*
```

Comment:
Please don't call strip in recipe. If you wanted to strip out debugging information, please use `defineDebugInfoPackage`. See [here] for an example.

[here]: https://github.com/haikuports/haikuports/wiki/HaikuPorter-BuildRecipes#a-simple-example-using-the-recommended-style-for-description